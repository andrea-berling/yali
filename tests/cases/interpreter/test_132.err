[line 8] Error at 'Dinosaur': Undefined reference.
