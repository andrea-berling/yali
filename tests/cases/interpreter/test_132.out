Inside block: Dinosaur exists
Dinosaur
Accessing out-of-scope class:
